README.md

```
@@ -109,6 +109,8 @@ docker compose up -d
 
 ## ER Diagram
 
+[![](./mini_scholl_management.png)]
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)

(MD045, no-alt-text)